packages/local/package.json

```diff
@@ -27,7 +27,8 @@
     "mocha": "^5",
     "nyc": "^13",
     "ts-node": "^10",
-    "typescript": "3.3.3333"
+    "tslib": "^1.9.3",
+    "typescript": "3.7.5"
```

Comment:
Aren't we upgrading typescript to 4.8.4? That's what it is everywhere else.

Reply:
This was the one place I needed to specify a package in order for tests to pass. It shouldn't have worked without this, but yarn 1 does what it wants. The TS upgrade will happen in the oclif-core-migration branch.